### PR TITLE
ICMSLST-720 Implement "resume application" link in workbasket.

### PR DESCRIPTION
**Pull request by oskusalerma**

### CODE REVIEW

**Review by kevinetienne — APPROVED** [review]

File: web/domains/case/export/views.py [hunk 1]
```diff
@@ -119,7 +119,9 @@ def edit_com(request, pk):
             if form.is_valid():
                 form.save()
 
-                return redirect(reverse("export:com-edit", kwargs={"pk": pk}))
+                return redirect(
+                    reverse("export:com-edit", kwargs={"application_pk": application_pk})
```
Comment (MattHolmes123) [inline web/domains/case/export/views.py:123]: shouldn't this be: `appl.get_edit_view_name()`

File: web/domains/case/export/views.py [hunk 2]
```diff
@@ -149,7 +151,8 @@ def submit_com(request, pk):
 
         errors = ApplicationErrors()
         page_errors = PageErrors(
-            page_name="Application details", url=reverse("export:com-edit", kwargs={"pk": pk})
+            page_name="Application details",
+            url=reverse("export:com-edit", kwargs={"application_pk": pk}),
```
Comment (MattHolmes123) [inline web/domains/case/export/views.py:155]: same here

File: web/domains/case/export/views.py [hunk 3]
```diff
@@ -161,7 +164,7 @@ def submit_com(request, pk):
             go_back_to_edit = "_edit_application" in request.POST
 
             if go_back_to_edit:
-                return redirect(reverse("export:com-edit", kwargs={"pk": pk}))
+                return redirect(reverse("export:com-edit", kwargs={"application_pk": pk}))
```
Comment (MattHolmes123) [inline web/domains/case/export/views.py:167]: and here

File: web/tests/domains/case/export/test_views.py [hunk 4]
```diff
@@ -34,11 +34,11 @@ def test_flow(self):
         self.login_with_permissions(["exporter_access"])
         assign_perm("web.is_contact_of_exporter", self.user, appl.exporter)
 
-        url_edit = reverse("export:com-edit", kwargs={"pk": appl.pk})
+        url_edit = reverse("export:com-edit", kwargs={"application_pk": appl.pk})
```
Comment (MattHolmes123) [inline web/tests/domains/case/export/test_views.py:37]: and all places in this test file

**Review by MattHolmes123 — CHANGES_REQUESTED** [review]

File: web/domains/case/export/views.py (hunk 1 above)
Comment (oskusalerma) [reply web/domains/case/export/views.py:123]: no, this is a specific function, edit_com

File: web/domains/case/export/views.py (hunk 2 above)
Comment (oskusalerma) [reply web/domains/case/export/views.py:155]: also a specific function, submit_com

File: web/domains/case/export/views.py (hunk 3 above)
Comment (oskusalerma) [reply web/domains/case/export/views.py:167]: still submit_com

File: web/tests/domains/case/export/test_views.py (hunk 4 above)
Comment (oskusalerma) [reply web/tests/domains/case/export/test_views.py:37]: all specific

**Review by MattHolmes123 — APPROVED** [review]
:shrug:

**Pull request merged**